frontend/src/organisms/ContactInfoForm.js

```diff
@@ -114,7 +78,8 @@ function ContactInfoForm() {
                   fullWidth
                   variant="contained"
                   color="primary"
-                  disabled={loading}
+                  disabled={loading || Object.values(formikBag.values).some(value => !value)}
```

Comment:
To `Object.values` muzes klidne taky napsate nekam zvlast. Klidne v te casti **constants** vytvor neco na zpracovatni tech objektu.

Reply:
Jo to mám v plánu, jen jsem už nestíhal, tak jsem to tam zapsal, abych nezapomněl.